res/controllers/Numark-Scratch.midi.xml

```diff
@@ -1,12 +1,102 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.3.0+">
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.4.0+">
```

Comment:
Should be `2.5.0` (I believe it is wrong on the S4 Mk3)

Reply:
Done